docs/installation.md

```diff
@@ -261,3 +261,12 @@ visible to the classloader which currently locates the javac Main class. Then
 replace the call of `javac.main.Main.main()` with the Error Prone compiler:
 
 `return new ErrorProneCompiler.Builder().build().compile(args) == 0`
+
+## Using ErrorProne in concert with other Annotation Processors
+
+All of the above instructions use the `javac` option `-processorpath` which has
+side-effect of causing `javac` to no longer scan the compile classpath for
+annotation processors. If you are using other annotation processors in addition
+to ErrorProne, such as Lombok, then you will need to add their JARs to your
```

Comment:
…or Dagger, AutoValue, Mapstruct, Immutables, etc.

Reply:
Of course. 
Are you asking me to change the PR to list these other examples?
I don't think that adds a lot, but I am happy to do so if you like.
Are you happy with listing those 5 examples? It seems a bit overkill to me